src/partials/sections/header.hbs

```diff
@@ -98,113 +100,104 @@
 	<nav class="mobile-nav position-fixed d-lg-none sni sni__close">
 		<ul id="mobileMenu" class="mobile-nav__menu accordion position-fixed h-100 col-9 bg-white list-unstyled py-2 mb-0"
 			role="tablist" aria-multiselectable="true">
-			<li class="py-1">
+			<li>
 				<a href="#" class="btn btn-primary btn-block">Shop All</a>
```

Comment:
please add additional margin-bottom as in the LIVE version. 
also vertical align the X close button middle to the the Shop All button.